sunkit_pyvista/plotter.py

```diff
@@ -37,14 +37,14 @@ class SunpyPlotter:
     all_meshes : `dict`
         Stores a reference to all the plotted meshes in a dictionary.
     """
+
     def __init__(self, coordinate_frame=None):
         if coordinate_frame is None:
             coordinate_frame = HeliocentricInertial()
         self._coordinate_frame = coordinate_frame
         self._plotter = pv.Plotter()
         self.camera = self._plotter.camera
         self.all_meshes = {}
-        self.color_maps = {v: k for k, v in cm.cmlist.items()}
```

Comment:
@jeffreypaul15 do you remember what the reason for this was? I had to remove it to get the doc build working, and wanted to check what it was for.

Reply:
This was to make sure that we're saving the name of the colormap when save is called.
The change was made in #55.
`map.cmap.name` gives us the names of the colormaps but we need their variable names, `cmlist` allowed has the mapping of variable names as keys and actual names as values so I created an inverse of cmlist to allow specification of the correct colormap when loading as we would be specifying the name and we'd need the variable name.

Reply:
Thanks - I've edited this a bit in this PR to make it more general and work with any Matplotlib colormap. Previously it was only working with sunpy colormaps.